packages/forms-web-app/src/pages/projects/project-information/includes/get-updates.njk

```diff
@@ -0,0 +1,24 @@
+<section>
+	<h2 class="govuk-heading-m">
+		Get updates
+	</h2>
+
+	<p class="govuk-body">Enter your email address to receive:</p>
+
+	<ul class="govuk-list govuk-list--bullet">
+		<li>updates on project progress</li>
+
+		<li>information on how to have your say</li>
+
+		<li>notifications when key documents are published</li>
+	</ul>
+
+	<div class="govuk-inset-text">
+		Read the <a
```

Comment:
Are we putting rel=”noopener noreferrer” on links? I don't think we have been so far, maybe something to look into and talk about: https://www.elegantthemes.com/blog/wordpress/rel-noopener-noreferrer-nofollow#:~:text=The%20rel%3D%E2%80%9Dnoopener%20noreferrer%E2%80%9D%20HTML%20attributes%20appear%20in%20external,in%20the%20destination%20site's%20analytics